tsup.config.ts

```diff
@@ -19,8 +19,33 @@ export default defineConfig([
     treeshake: true,
     globalName: "ThirdwebSDK",
     format: ["cjs", "esm"],
+    outDir: "dist/node",
+    banner: {
+      js: `import "cross-fetch/dist/node-polyfill.js";`,
```

Comment:
importing the cress-fetch polyfill only in the node case (this is injected onto the top of the file)